openrouteservice/src/main/java/heigit/ors/api/requests/routing/RouteRequest.java

```diff
@@ -227,6 +227,14 @@
     @JsonIgnore
     private boolean hasSimplifyGeometry = false;
 
+    @ApiModelProperty(name = "skip_segments", value = "Specifies the segments that should be skipped in the route calculation. " +
```

Comment:
You should create a constant that contains the "skip_segments" text and then reference that